src/components/NewComps/AffiliateForm/AffiliateForm.module.scss

```
@@ -0,0 +1,101 @@
+
+.submitBtn {
+  height: 52px;
+  border-radius: 12px;
+}
+
+.otpInput {
+  width: 48px;
+}
+
+.phoneField {
+  :global(.iti) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)